features/step_definitions/membership_application_steps.rb

```diff
@@ -17,7 +17,7 @@
   visit edit_membership_application_path(membership_application)
 end
 
-Given(/^I am on "([^"]*)" application page$/) do |first_name|
+Given(/^I am on (the show )+"([^"]*)" application page$/) do |optional, first_name|
```

Comment:
I know, houndbot.  But it's a regular expression and I don't know how to deal with it any better than that right now.